pkg/oci/remote/signatures.go

```diff
@@ -49,6 +52,42 @@ func Signatures(ref name.Reference, opts ...Option) (oci.Signatures, error) {
 	}, nil
 }
 
+func Bundle(ref name.Reference, opts ...Option) (*sgbundle.Bundle, error) {
+	o := makeOptions(ref.Context(), opts...)
+	img, err := remoteImage(ref, o.ROpt...)
+	if err != nil {
+		return nil, err
+	}
+	// TODO: We can check for a specific predicate type here using
```

Comment:
To clarify, do you mean checking for `application/vnd.dev.sigstore.bundle`? If so, I think hardcoding this is fine and we don't need this TODO.

Reply:
I'll remove the TODO as it's just an optimization suggestion -- the manifest pointing to the bundle is annotated with the in-toto predicate type, and we can filter it here so it doesn't have to download, parse, and verify every bundle.